lib/instance.js

```diff
@@ -22,10 +22,9 @@ var initValues = function(values, options) {
     defaults = {};
 
     if (this.Model._hasDefaultValues) {
-      Utils._.each(this.Model._defaultValues, function(valueFn, key) {
-        if (defaults[key] === undefined) {
-          defaults[key] = valueFn();
-        }
+      defaults = _.mapValues(this.Model._defaultValues, function(valueFn) {
+        var value = valueFn();
+        return value._isSequelizeMethod ? value : _.cloneDeep(value);
```

Comment:
I tried `_.clone` and `_.cloneDeep` before, but that would cause an issue with sequelize methods, making an integration test fail and potentially introducing regressions. More info in https://github.com/seegno-forks/sequelize/pull/3#issuecomment-166697719 and https://github.com/seegno-forks/sequelize/pull/3#issuecomment-167829881